participants/FilevipCloud.md

```diff
@@ -0,0 +1,77 @@
+# <FilevipCloud>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Registration Submission, Project Name, and Team Slack Handles
+
+*The Submission ID*: *666418435*
+
+Project Name: FilevipCloud
+
+Development team:  Star Public Chain Tech Dept
+
+## (1) Brief Project Description
+
+*Devote to creating an exquisite, elegant and simple cloud disk based on the IPFS network and the Filecoin network.*
+
+## (2) Link to Project UI
+
+http://cloud.filevip.top:8089/
+
+## (3) What does your application/UI do?
+
+*document list
+File/folder upload + drag and drop upload
+File sharing
+File monitoring statistics
+Recycle bin
+Multi-user
+WebDav
+Scan disk task
+Online preview and custom configuration preview engine*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*I don’t understand what curated dataset is. We are for Multi-users to upload any data to the IPFS network and Filecoin network.*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+
+*What kind of data is collected depends on the user or the company. We will add audit functions in the future, similar to [百度云盘](https://yun.baidu.com/).The review of illegal data in China is very strict. We need to protect users' personal data while ensuring that there is no illegal data.*
```

Comment:
Please ensure that there is a way for the evaluation committee to retrieve the data (i.e. via an API or key) so that the contents for the competition can be validated.